pyrometheus/chem_expr.py

```diff
@@ -164,8 +167,63 @@ def _zeros_like(argument):
 
 # }}}
 
-# {{{ Equilibrium constants
 
+# {{{ Transport polynomials & mixture rules
+
+def transport_polynomial_expr(c, n, t):
+    """Generate code for transport polynomials
+
+    :returns: Transport polynomial expression with coefficients c in terms of
+    the temperature t as a :class:`pymbolic.primitives.Expression`. For
+    `thermal_conductivity`, `n = 1`, while for `viscosity` `n = 2`
+    """
+    assert len(c) == 5
+    return (
+        (
+            c[0]
+            + c[1] * p.Variable("log")(t)
+            + c[2] * p.Variable("log")(t) ** 2
+            + c[3] * p.Variable("log")(t) ** 3
+            + c[4] * p.Variable("log")(t) ** 4
+        )**n
+    )
+
+
+def viscosity_mixture_rule_wilke_expr(sol: ct.Solution, sp, x, mu):
+    """Generate code for species mixture rule.
+       See Robert J. Kee, Michael E. Coltrin and Peter Glarborg
+       "Chemically Reacting Flow" book, chapter 12.
```

Comment:
Fix indentation. (Should line up with rest of code.)

Reply:
Removed these two lines and added the proper citation.